src/client/game.cpp

```diff
@@ -2286,10 +2286,11 @@ void Game::toggleFog()
 
 void Game::toggleDebug()
 {
-	// Initial / 4x toggle: Chat only
+	// Initial / 3x toggle: Chat only
 	// 1x toggle: Debug text with chat
 	// 2x toggle: Debug text with profiler graph
 	// 3x toggle: Debug text and wireframe
+	// 4x toggle: Store inventory images (if enabled) & go back to chat
```

Comment:
honestly i'm not for using debug to dump images, it's not the good way to do it, and you will just dump by showing debug by error many times while switching.

Reply:
It will only work if the setting is set to something > 0, which it isn't by default. But I agree that I should find a better way, which is why this still is WIP. Suggestions?